test/nix/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG IMAGE=renovate/buildpack
+FROM ${IMAGE} as build
+
+# renovate: datasource=github-releases lookupName=NixOS/nix
+RUN install-tool nix 2.3.10
+
+USER ubuntu
+
+RUN set -ex; \
+  nix --version
+
+RUN set -ex; \
+  nix-instantiate --eval -E '(import <nixpkgs> {}).lib.version'
+
+RUN set -ex; \
```

Comment:
What are the commands renovate needs to run later? We only need to test those.

Reply:
Left the first test, and switched out the 2nd one for the command renovate will need to run to get the existing version.